CommonControls/Interfaces/AssetManagement/AssetConfigData.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommonControls.FileTypes.Animation;
+using CommonControls.FileTypes.PackFiles.Models;
+using CommonControls.FileTypes.RigidModel;
+using CommonControls.FileTypes.WsModel;
+
+namespace CommonControls.Interfaces.AssetManagement
+{
+    public class AnimationSettingStruct
+    {
+        public float Fps { get; set; }
+        public float FrameCount { get; set; }
+        public float TimeEnd { get; set; }
+    }
+
+    enum UnitEnum
```

Comment:
Why do you want to expose this to the user? 
Make it simple - always export using centemters or something. More options are not always a good thing

Reply:
This structure, is meant to be set from the export/import UI, (people want those settings for animation import/export)
How else should I do that?

Reply:
Yes ok, I will encapsulate more complelyt.